stdlib/nodejs/dir.rb

```diff
@@ -1,16 +1,18 @@
 class Dir
   @__glob__ = `require('glob')`
   @__fs__ = `require('fs')`
+  @__path__ = `require('path')`
   `var __glob__ = #{@__glob__}`
   `var __fs__ = #{@__fs__}`
+  `var __path__ = #{@__path__}`
 
   class << self
     def [](glob)
       `__glob__.sync(#{glob})`
     end
 
     def pwd
-      `process.cwd()`
+      `__path__.normalize(process.cwd()).split(__path__.sep).join(__path__.posix.sep)`
```

Comment:
I don't think `normalize` is mandatory here

Reply:
remove it 👍, from [node's docs]() seems it just removes double dots, and [we don't know if cwd() can have them](https://nodejs.org/api/process.html#process_process_cwd), so it shouldn't make a difference in correctness, on the contrary it might be that `.cwd()` is more close to `.pwd`.